version

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
Why do we need a version file, when we have a changelog already, where the version is logged?